server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/executor/ClusterConfigTaskExecutor.java

```diff
@@ -270,60 +269,71 @@ public SettableFuture<ConfigTaskResult> setTTL(SetTTLStatement setTTLStatement,
   }
 
   @Override
-  public SettableFuture<ConfigTaskResult> merge(TMergeReq tMergeReq) {
+  public SettableFuture<ConfigTaskResult> merge(boolean isCluster) {
     SettableFuture<ConfigTaskResult> future = SettableFuture.create();
+    TSStatus tsStatus = new TSStatus();
+    if (isCluster) {
+
+    } else {
+      tsStatus = LocalConfigNode.getInstance().executeMergeOperation();
+    }
     try (ConfigNodeClient client =
         CONFIG_NODE_CLIENT_MANAGER.borrowClient(ConfigNodeInfo.partitionRegionId)) {
       // Send request to some API server
-      TSStatus tsStatus = client.merge(tMergeReq);
-      // Get response or throw exception
-      if (tsStatus.getCode() == TSStatusCode.SUCCESS_STATUS.getStatusCode()) {
-        future.set(new ConfigTaskResult(TSStatusCode.SUCCESS_STATUS));
-      } else {
-        future.setException(new StatementExecutionException(tsStatus));
-      }
+      tsStatus = client.merge();
     } catch (IOException | TException e) {
       future.setException(e);
     }
```

Comment:
Why this block of code doesn't inside the if statement like flush and clearCache does?